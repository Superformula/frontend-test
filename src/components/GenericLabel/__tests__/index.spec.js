import React from 'react';
import { mount } from 'enzyme';
import { GenericLabel } from '../';

describe('GenericLabel', () => {
  it('should render without failing', () => {
    const wrapper = mount(<GenericLabel>Lorem ipsum</GenericLabel>);
    expect(wrapper).toMatchSnapshot();
  });
});
