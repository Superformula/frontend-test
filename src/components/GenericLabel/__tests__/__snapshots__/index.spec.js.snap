// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenericLabel should render without failing 1`] = `
.c0 {
  text-transform: capitalize;
  font-size: 12px;
  font-weight: 400;
  color: #757575;
}

<GenericLabel>
  <span
    className="c0"
  >
    Lorem ipsum
  </span>
</GenericLabel>
`;
